ferrilata-devilline/appsettings.json

```diff
@@ -1,8 +1,8 @@
-{
-  "Logging": {
-    "LogLevel": {
-      "Default": "Warning"
-    }
-  },
-  "AllowedHosts": "*"
+﻿{
+    "Logging": {
+        "LogLevel": {
+            "Default": "Warning"
+        }
+    },
+    "00": "*"
```

Comment:
Any reason why allowed hosts were changed to 00?

Reply:
This is some VS automatic mumbo jumbo